recipes/plant_tribes_gene_family_classifier/0.1/build.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+mkdir -p $PREFIX/bin
+mkdir -p $PREFIX/config
+
+cp ./pipelines/GeneFamilyClassifier $PREFIX/bin
+cp ./config/plantTribes.config $PREFIX/config
```

Comment:
I'm just wondering if the config file is used? How does the perl script find the config file?
Feel free to merge!